### ISSUE TITLE
"Could not load 'Autodesk.Revit.DB.EvaluatedParmeter' " when using Revit Lookup

### ISSUE THREAD

**Issue by InkRF** [jeremytammik/RevitLookup#160]
Greetings all,

I'm getting a issue that whenever I try to use a feature within RevitLookup I get the below error:

![image](https://user-images.githubusercontent.com/74789109/234660572-bb6d18b8-7124-41b8-817a-40dcd96fe143.png)

I'm using Revit 2022 on Windows 10, and while I swear I saw it stated somewhere that it is supported, is this error a result of the current version of Revit Lookup not working in Revit 2022? I see that the EvaluatedParameter class is new in the 2024 version of the Revit API so that would make sense. 

Thanks

**Comment by Nice3point** [comment]
EvaluatedParameter has been added in Revit version 2024. It is not supported in version 22. Did you manually move the plugin files ? Please use the installer for version 2022

![изображение](https://user-images.githubusercontent.com/20504884/234662849-dc05fb53-7f32-4dc5-b08f-f8ab11c9e354.png)

**Comment by InkRF** [comment]
Well, I guess all I can say is

![](https://media4.giphy.com/media/v1.Y2lkPTc5MGI3NjExMTllYzIzMWExMjVkOTRkMGZjMTY5NmI2MmFjMDEzZjc2Y2UwYjA1YiZlcD12MV9pbnRlcm5hbF9naWZzX2dpZklkJmN0PWc/3xz2BLBOt13X9AgjEA/giphy.gif)

I had though those 2021, 2022, 2023 were past releases of RevitLookup, not the current release for RevitLookup LOL

Thanks @Nice3point for the work you've put into this, I look forward to seeing what cool new things it has to offer!

**Comment by Nice3point** [comment]
Thank you very much) 
Links to the most recent RevitLookup versions for each Revit version are listed here:
https://github.com/jeremytammik/RevitLookup/wiki/Versions

@jeremytammik  Hi Jeremy, can you add this link in any of the releases on your blog ? I think there are those who think RevitLookup is only available for version 2024, although it was mentioned in one of the last updates

**Comment by jeremytammik** [comment]
Yes, of course! With pleasure. I removed the blog post notice of 2024.0.3 and the debug tab. Let's get a new post out there asap with 2024.0.4, and add an explanation about support for previous Revit versions to that. Currently, 2024.0.2 shows up as the latest release. Will you update that to 2024.0.4 later today? Sorry for the urgency, but I got a tremendous amount of pressure internally overnight. Luckily, I was completely offline, so I only found out this morning...
